pkgs/data/fonts/lxgw-wenkai/default.nix

```
@@ -1,17 +1,22 @@
 { lib, fetchzip }:
 
-let version = "1.222"; in
+let
+  version = "1.245.1";
+in
 fetchzip {
   name = "lxgw-wenkai-${version}";
```

```suggestion
fetchzip rec {
  pname = "lxgw-wenkai";
  version = "1.245.1";
```
This should work since recently

---

The new format looks nice.